packages/betterer/src/files/file-info.ts

```diff
@@ -66,7 +66,9 @@ export class BettererFileInfo {
           printed += ',\n';
         }
         const [line, column, length, message] = mark;
-        printed += `      [${line}, ${column}, ${length}, "${message}"]`;
+        printed += `      [${line}, ${column}, ${length}, ${JSON.stringify(
```

Comment:
First stop

1. We know this output is eventually going to be sent through `JSON.parse`. There's a lot of characters that can make a string invalid, just as other `"` marks, new lines, etc.

👉 [Continue]()